mkdocs.yml

```diff
@@ -11,6 +11,7 @@ validation:
   omitted_files: warn
   absolute_links: warn
```

Comment:
MkDocs `1.6.0` has a new `absolute_links: relative_to_docs` setting. We currently use `absolute_links: warn`. Please read the [MkDocs User Guide: validation of absolute links](https://www.mkdocs.org/user-guide/configuration/#validation-of-absolute-links) to learn about the new option. It's a bit difficult for me to understand what the new option does, and if we should change to the new option, or keep what we have now. :upside_down_face:

Reply:
yes, let's do that. my PR validates it on our docs and `/...` links are now forbidden. also `https://docs.renovateboit.com/...` links are now forbidden

Reply:
The way I'm reading the MkDocs section about validation of absolute links:

- `absolute_link: warn` blocks us from using _absolute_ links
- MkDocs says that only using _relative_ links is a good convention to follow, as you can browse the source pages without MkDocs
- If you leave absolute links in the source files, thing may not work as expected, or as we have set things up now: MkDocs warns us about the absolute link
- If you want to also use/allow absolute links, then you can use the `relative_to_docs` setting
- I think we still only want to allow _relative_ links, and block/warn for _absolute_ links?

I'm not sure what you mean with "yes, let's do that", do you mean:

1. Change to `absolute_links: relative_to_docs`
2. Keep `absolute_links: warn`

Full quote of the relevant part of the MkDocs User Guide:

> #### Validation of absolute links
> 
> NEW: **New in version 1.6.**
> 
> > Historically, within Markdown, MkDocs only recognized **relative** links that lead to another physical `*.md` document (or media file). This is a good convention to follow because then the source pages are also freely browsable without MkDocs, for example on GitHub. Whereas absolute links were left unmodified (making them often not work as expected) or, more recently, warned against. If you dislike having to always use relative links, now you can opt into absolute links and have them work correctly.
> 
> If you set the setting `validation.links.absolute_links` to the new value `relative_to_docs`, all Markdown links starting with `/` will be understood as being relative to the `docs_dir` root. The links will then be validated for correctness according to all the other rules that were already working for relative links in prior versions of MkDocs. For the HTML output, these links will still be turned relative so that the site still works reliably.
> 
> So, now any document (e.g. "dir1/foo.md") can link to the document "dir2/bar.md" as `[link](/dir2/bar.md)`, in addition to the previously only correct way `[link](../dir2/bar.md)`.
> 
> You have to enable the setting, though. The default is still to just skip the link.
> 
> > EXAMPLE: **Settings to recognize absolute links and validate them:**
> >
> > ```yaml
> > validation:
> >   links:
> >     absolute_links: relative_to_docs
> >     anchors: warn
> >     unrecognized_links: warn
> > ```
>

Reply:
ok, let's leave to warn and don't use them. i block them now anyways on our new docs tersts